apps/bfd-pipeline/bfd-pipeline-rda-grpc-server/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/DirectRdaLoadApp.java

```diff
@@ -0,0 +1,67 @@
+package gov.cms.bfd.pipeline.rda.grpc.sink;
+
+import com.codahale.metrics.MetricRegistry;
+import gov.cms.bfd.pipeline.rda.grpc.RdaLoadJob;
+import gov.cms.bfd.pipeline.rda.grpc.RdaLoadOptions;
+import gov.cms.bfd.pipeline.rda.grpc.source.GrpcRdaSource;
+import gov.cms.bfd.pipeline.sharedutils.DatabaseOptions;
+import gov.cms.bfd.pipeline.sharedutils.DatabaseUtils;
+import gov.cms.bfd.pipeline.sharedutils.IdHasher;
+import gov.cms.bfd.pipeline.sharedutils.PipelineJob;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.Reader;
+import java.time.Duration;
+import java.util.Properties;
+
+/**
+ * Simple application that invokes an RDA API server and writes FISS claims to a database using
+ * RdaLoadJob. This program was written for testing purposes and is not planned to be executed in a
+ * production environment.
+ */
+public class DirectRdaLoadApp {
```

Comment:
So if I understand this correctly, this is just one use case to run unit tests on the server to make sure its working correctly.  We could create another LoadApp that runs the server on a process independent of the RDA client?  If for example we wanted to test that the two services can communicate over network calls?

Reply:
Well this is a simple program that just connects to an RDA API server and dumps data to the database using the normal pipeline job.  As such it's great for testing the job without having to bring up a PipelineApp instance anywhere.  It could also be useful when we want to populate a test or sandbox database using the mock RDA API server since we could just run it locally with a connection to the database we are populating.